package.json

```diff
@@ -113,7 +113,8 @@
   "scripts": {
     "build": "node scripts/build/package.js",
     "clean": "node scripts/build/clean.js",
-    "format": "npm run prettier -- --write && npm run xo:fix",
+    "format:icondata": "node scripts/format-icon-data.js",
+    "format": "npm run format:icondata && npm run prettier -- --write && npm run xo:fix",
```

Comment:
What's the benefit in splitting the icon data into its own script here? I can't see myself ever using it separately from format.

Reply:
To make our scripts consistent. Like `xo:fix` does. The change was requested by @mondeja.

Reply:
When you separate npm scripts and run them from another one, npm shows in their output each `npm run` command executed. If a failure happens or the command takes some apreciable time, it's easy to know what script is causing it.